influxdb3/tests/server/configure.rs

```diff
@@ -1049,14 +1049,9 @@ async fn api_v3_configure_table_create_then_write() {
         .expect("create table call failed");
     assert_eq!(StatusCode::OK, resp.status());
     let result = server
-        .api_v3_query_sql(&[
-            ("db", "foo"),
-            ("q", "SELECT * FROM bar"),
-            ("format", "json"),
-        ])
-        .await
-        .json::<Value>()
-        .await
+        .query_sql("foo")
```

Comment:
nit: The semantics of this bother me a bit it reads as "query <database> with query" which feels weird rolling off the mental togue. What do you think about `.query(<database_name>).with_sql(<sql_string>)` instead? Then for influxql queries we would have `.query(<database_name>).with_influxql(<influxql_string>)`?

Reply:
Actually, I just realize I was confusing myself -- I thought this change was happening in the `influxdb3_client` crate and was hoping to see an improvement in the semantics of that API :facepalm:. Thanks for making the requested change!